### DISCUSSION TITLE
A collection for random unplayed movies

### DISCUSSION THREAD

**Discussion by cometandluna** [meisnate12/Plex-Meta-Manager#272] [Q&A]
My idea was to create a Collection in Plex that would find all of my unplayed movies above a certain audience rating, and then randomly put 10 of those movies into the Collection.  This is what I came up with for a Collection in my movies.yml file:

```
Random Unwatched Movies:
    smart_filter:
      all:
        audience_rating.gte: 7.0
        plays.lt: 1
      sort_by: random
      limit: 10
```

I'm trying this out as a Smart Filter.  What actually comes up is a list of 10 movies that are almost all unplayed that do match the minimum audience rating I listed.

As far as results from my original idea, this is pretty close, but I would like it better.  Because I couldn't find a filter for Unplayed movies in Smart Filters, I'm using play.lt to 1, so any movie with 0 full plays is part of the filter.  I did notice movies that In Progress movies showing up in the list, which I would prefer not to.  The other issue is that, every time I go into the collection, it displays another different set of random movies.  I would prefer this list to refresh (or re-randomize) only when I run Plex Meta Manager, which I currently run the docker container run once daily.

Is there any way I can make this work as a collection closer to my original idea?

**Comment by meisnate12** [comment]
i can add unplayed as a smart filter option to help you there in terms of making it more random I'm afraid that a plex issue it should randomize each day but I haven't figured out a way to trigger it randomizing if you do let me know and ill see what I can do

**Reply by cometandluna** [reply]
If you could add an unplayed Smart Filter, that would be great.  In terms of the "randomness", I only want it to refresh that random list when I run Plex Meta Manager (once a day for me), and not every time I go back into that collection.

**Reply by meisnate12** [reply]
Unfortunately I'm most likely not going to be able to do anything about the randomness aspect of this you could try it as a smart label with a plex search for unwatched which I'm also going to add but I don't have random sort there I can test out random there and see if it works and add it as well I'll let you know

**Reply by meisnate12** [reply]
unplayed has been added to `smart_filter` and `plex_search`
